pkg/dinghyfile/builder.go

```diff
@@ -311,6 +311,10 @@ func (b *PipelineBuilder) RebuildModuleRoots(org, repo, path, branch string) err
 	b.RebuildingModules = true
 	errEncountered := false
 	failedUpdates := []string{}
+	if b.Action == pipebuilder.Validate {
+		// Since we are checking for modules, those live in master
+		branch = "master"
```

Comment:
Will we see issues with this approach if the primary branch is `main`?

Reply:
Since we have the logic for master/main in the Downloader we are safe, it will check for main if master doesn't exists.